diff --git a/node_modules/vuetify/dist/vuetify-labs.esm.js b/node_modules/vuetify/dist/vuetify-labs.esm.js
index 90a440b..169a861 100644
--- a/node_modules/vuetify/dist/vuetify-labs.esm.js
+++ b/node_modules/vuetify/dist/vuetify-labs.esm.js
@@ -5350,6 +5350,7 @@ function rippleShow(e) {
     if (element._ripple.isTouch) return;
   }
   value.center = element._ripple.centered || isKeyboardEvent(e);
+  ripples.hide(element);
   if (element._ripple.class) {
     value.class = element._ripple.class;
   }
diff --git a/node_modules/vuetify/dist/vuetify-labs.js b/node_modules/vuetify/dist/vuetify-labs.js
index 6fd9654..62bedf6 100644
--- a/node_modules/vuetify/dist/vuetify-labs.js
+++ b/node_modules/vuetify/dist/vuetify-labs.js
@@ -5354,6 +5354,7 @@
       if (element._ripple.isTouch) return;
     }
     value.center = element._ripple.centered || isKeyboardEvent(e);
+    ripples.hide(element);
     if (element._ripple.class) {
       value.class = element._ripple.class;
     }
diff --git a/node_modules/vuetify/dist/vuetify.esm.js b/node_modules/vuetify/dist/vuetify.esm.js
index b889451..ae99fce 100644
--- a/node_modules/vuetify/dist/vuetify.esm.js
+++ b/node_modules/vuetify/dist/vuetify.esm.js
@@ -5583,6 +5583,7 @@ function rippleShow(e) {
     if (element._ripple.isTouch) return;
   }
   value.center = element._ripple.centered || isKeyboardEvent(e);
+  ripples.hide(element);
   if (element._ripple.class) {
     value.class = element._ripple.class;
   }
diff --git a/node_modules/vuetify/dist/vuetify.js b/node_modules/vuetify/dist/vuetify.js
index c97b294..25abf22 100644
--- a/node_modules/vuetify/dist/vuetify.js
+++ b/node_modules/vuetify/dist/vuetify.js
@@ -5587,6 +5587,7 @@
       if (element._ripple.isTouch) return;
     }
     value.center = element._ripple.centered || isKeyboardEvent(e);
+    ripples.hide(element);
     if (element._ripple.class) {
       value.class = element._ripple.class;
     }
diff --git a/node_modules/vuetify/lib/directives/ripple/index.mjs b/node_modules/vuetify/lib/directives/ripple/index.mjs
index a060ba9..45de0e2 100644
--- a/node_modules/vuetify/lib/directives/ripple/index.mjs
+++ b/node_modules/vuetify/lib/directives/ripple/index.mjs
@@ -132,6 +132,7 @@ function rippleShow(e) {
     if (element._ripple.isTouch) return;
   }
   value.center = element._ripple.centered || isKeyboardEvent(e);
+  ripples.hide(element);
   if (element._ripple.class) {
     value.class = element._ripple.class;
   }
